src/color/color_utils.js

```diff
@@ -11,22 +11,24 @@ define(function(require) {
   /**
    * For a color expressed as an HSBA array, return the corresponding RGBA value
    * 
-   * @param {Array} hsba An 'array' object that represents a list of
-   *                          HSB colors on a scale of 0-255
+   * @param {Array} hsba An 'array' object that represents a list of HSB colors
+   * @param {Array} maxes An 'array' object that represents the HSB range maxes
    * @return {Array} an array of RGBA values, on a scale of 0-255
    */
-  p5.ColorUtils.hsbaToRGBA = function(hsba) {
+  p5.ColorUtils.hsbaToRGBA = function(hsba, maxes) {
     var h = hsba[0];
     var s = hsba[1];
     var v = hsba[2];
-    h /= 255;
-    s /= 255;
-    v /= 255;
+    var a = hsba[3] || maxes[3];
+    h /= maxes[0];
+    s /= maxes[1];
+    v /= maxes[2];
+    a /= maxes[3];
```

Comment:
I think this is a good time to open an issue about style. We have a link to [idiomatic.js](https://github.com/rwaldron/idiomatic.js/) in the developer doc but I think it's sort of long and doesn't explicitly cover a lot of things (like this case for example). I'll open one now. I suggest we leave this part as is for now until we have some clearer guide.